<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="load_status" native="true">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="cpu_status" native="true">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="memory_status" native="true">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="disk_status" native="true">
   <property name="geometry">
    <rect>
     <x>609</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="disk_space">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>205</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0G/0G</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>73</y>
     <width>16</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>73</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU使用率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="cpu_core_num">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>205</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0核心</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="load_status_info">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>205</y>
     <width>56</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>运行流畅</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>73</y>
     <width>56</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>负载状态</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="memory_space">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>205</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0/0(MB)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="Dtk::Widget::DLabel_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>73</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>内存使用率</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Dtk::Widget::DLabel" name="system_info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>disk_space</zorder>
  <zorder>Dtk::Widget::DLabel_8</zorder>
  <zorder>Dtk::Widget::DLabel_6</zorder>
  <zorder>cpu_core_num</zorder>
  <zorder>load_status_info</zorder>
  <zorder>Dtk::Widget::DLabel_9</zorder>
  <zorder>memory_space</zorder>
  <zorder>Dtk::Widget::DLabel_7</zorder>
  <zorder>disk_status</zorder>
  <zorder>memory_status</zorder>
  <zorder>cpu_status</zorder>
  <zorder>load_status</zorder>
  <zorder>system_info</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dtk::Widget::DLabel</class>
   <extends>QLabel</extends>
   <header>DLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
